services/client/src/pages/events/index.js

```diff
@@ -30,8 +32,13 @@ const mapDispatchToProps = function(dispatch) {
   };
 };
 
-const eventsUrl = `${process.env.REACT_APP_EVENTS_SERVICE_URL}/events`;
-const PaginatedEvents = withPagination(eventsUrl)(Events);
+const urlObject = {
+  url: `${process.env.REACT_APP_EVENTS_SERVICE_URL}/events`,
+  params: {
+    location: "dublin"
+  }
+};
+const PaginatedEvents = withPagination(urlObject)(Events);
```

Comment:
I think the `withPagination` component here is causing some additional complexity. I'm not sure how to best solve this so I'll go have a read on how other projects handle this but perhaps @alistairjcbrown has some ideas.

Reply:
Yeah, it's complex because we have to set different parameters in different places.